srtcore/buffer.cpp

```diff
@@ -196,19 +190,19 @@ void CSndBuffer::addBuffer(const char* data, int len, int ttl, bool order, uint6
     }
     m_pLastBlock = s;
 
-    CGuard::enterCS(m_BufLock);
+    CriticalSection::enter(m_BufLock);
```

Comment:
You have no unusual path here. Maybe better to mark a critical section block?

Reply:
Do you mean the `ScopedLock` (aka `CGuard`) inside a code block?